include/os/linux/spl/sys/taskq.h

```diff
@@ -84,6 +84,8 @@ typedef struct taskq {
 	int			tq_nthreads;	/* # of existing threads */
 	int			tq_nspawn;	/* # of threads being spawned */
 	int			tq_maxthreads;	/* # of threads maximum */
+	/* original maxthreads arg to create */
+	int			tq_orig_maxthreads;
```

Comment:
I think this is only relevant when using PER_CPU.  So it would be more clear to define this as "if PER_CPU flag is set, this is the number of threads is this percent of NCPUs".  And rename it to something like `tq_cpu_pct`.